spec/spec_helper.rb

```diff
@@ -1,8 +1,14 @@
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.